### ISSUE TITLE
Django Admin: Indicate whether sumitter has other domains

### ISSUE THREAD

**Issue by PaulKuykendall** [cisagov/manage.get.gov#1850]
### Issue description

Problem: an analyst would like to know whether the submitter is already associated with other domains in the system. This is an important indicator as to whether this submitter is trustworthy and established.

Solution: create an indicator to be located near the submitter name or email address, which shows how many other domains they are as associated with in our system. If possible, this could be a link to look at the other domains.

### Acceptance criteria

- [ ] an indicate near the submitter name, shows how many other domains in the system with which they are associated.

**Comment by vickyszuchin** [comment]
Cameron would like to extend the scope to include approved domains as well as domain applications. cc: @PaulKuykendall @abroddrick

**Comment by vickyszuchin** [comment]
Block tag add: this ticket is blocked by a ticket that is planned for sprint 41 (#1852)

**Comment by vickyszuchin** [comment]
"Blocked" tag removed. Dependency on #1852 is resolved.

**Comment by PaulKuykendall** [comment]
This ticket is already done, however, to clarify.. this should have been "creator" only, rather than "submitter."

Dave worked with Alysia to do the right thing (which I intended)... to do this for `users` (Creator) rather then `contacts` (Submitter).

So, it's done correctly, even tho the ticket definition was in error.
